packages/@aws-cdk/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -401,3 +418,11 @@ function encodeBooleans(object: object) {
     }
   });
 }
+
+function encodeAwsSdkCall(call: AwsSdkCall): EncodedAwsSdkCall {
```

Comment:
naming: `formatCall()` or `renderCall()`?

Then we can also handle the `undefined` case in the function instead of keeping the `props.Xxx && fn(Xxx)`?